app/controllers/v1/sessions/report_templates_controller.rb

```diff
@@ -67,6 +69,16 @@ def remove_logo
     render status: :ok
   end
 
+  def remove_cover_letter_custom_logo
+    authorize! :remove_cover_letter_custom_logo, report_template
+
+    return head :forbidden unless @session.ability_to_update_for?(current_v1_user)
+
+    report_template.cover_letter_custom_logo.purge
+
+    render status: :ok
+  end
+
```

Comment:
If you have bounced from a branch other than 'dev', then when creating the PR, the base should be set to this branch. Thanks to this, the checker will only see the real changes that were made as part of this task